sigs.yaml

```diff
@@ -958,7 +958,15 @@ sigs:
       day: Wednesday
       time: "9:30"
       tz: "PT (Pacific Time)"
-      frequency: weekly
+      frequency: weekly - except fourth Wednesday every month
+      url: https://docs.google.com/document/d/1FQx0BPlkkl1Bn0c9ocVBxYIKojpmrS1CFP5h0DI68AE/edit
```

Comment:
I guess we can avoid adding the meeting notes link again because it will be the same :)

Reply:
I agree. I was copying from other SIGs.

But I think the resulting markdown can be improved too if we remove these URLs. Do you think we should modify the structure to allow more than one time for the same meeting? In that case we can gernerate better markdown. e.g.

```
meetings:
- description: Regular SIG Meeting
  day: Wednesday
  time: "9:30"
  tz: "PT (Pacific Time)"
  frequency: weekly - except fourth Wednesday every month
  url: https://docs.google.com/document/d/1FQx0BPlkkl1Bn0c9ocVBxYIKojpmrS1CFP5h0DI68AE/edit
  archive_url: https://docs.google.com/document/d/1qf-02B7EOrItQgwXFxgqZ5qjW0mtfu5qkYIF1Hl4ZLI/
  recordings_url: https://www.youtube.com/watch?v=EMGUdOKwSns&list=PL69nYSiGNLP2x_48wbOPO0vXQgNTm_xxr
  alternatives:
  - description: APAC SIG Meeting
    day: Wednesday
    time: "20:00"
    tz: "PT (Pacific Time)"
    frequency: monthly - fourth Wednesday every month
```

or

```
meetings:
- description: SIG Meeting
  url: https://docs.google.com/document/d/1FQx0BPlkkl1Bn0c9ocVBxYIKojpmrS1CFP5h0DI68AE/edit
  archive_url: https://docs.google.com/document/d/1qf-02B7EOrItQgwXFxgqZ5qjW0mtfu5qkYIF1Hl4ZLI/
  recordings_url: https://www.youtube.com/watch?v=EMGUdOKwSns&list=PL69nYSiGNLP2x_48wbOPO0vXQgNTm_xxr
  times:
  - day: Wednesday
    time: "9:30"
    tz: "PT (Pacific Time)"
    frequency: weekly - except fourth Wednesday every month
  - day: Wednesday
    time: "20:00"
    tz: "PT (Pacific Time)"
    frequency: monthly - fourth Wednesday every month
```

Reply:
OK, I think this should be a follow-up.

@nikhita I've removed `archive_url` and `recordings_url` but left `url` to avoid an empty link. PTAL.